lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -90,6 +90,18 @@ class ViaqDataModelFilter < Filter
     desc 'Name of destination timestamp field'
     config_param :dest_time_name, :string, default: '@timestamp'
 
+    # <event_trans>
```

Comment:
Why do we call this `event_trans`?  Are you planning on having more transformations than just from `eventrouter`?  Perhaps this framework could be used to transform events from a docker daemon?

If we are not going to go this route, then having the extra `type` specifier seems redundant with the `event_trans` name itself.

Reply:
This could even be another `formatter`

Reply:
@portante you are correct, the plan is to perform transformations from docker events as well once we have them